public/data/dunnjacoba.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "Jacob Dunn",
+  "type": "personal",
+  "bio": "Freelance Dev, Open source beginner",
+  "avatar": "https://github.com/dunnjacoba.png",
+  "links": [
+    {
+      "name": "Check out my green boxes on Github!",
```

Comment:
I recommend capitalizing the `h` here. What do you think?
```suggestion
      "name": "Check out my green boxes on GitHub!",
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question